cmd/operator-sdk/alpha/cmd.go

```diff
@@ -25,6 +28,10 @@ func NewCmd() *cobra.Command {
 		Short: "Run an alpha subcommand",
 	}
 
-	cmd.AddCommand(olm.NewCmd())
+	cmd.AddCommand(
+		olm.NewCmd(),
+		up.NewCmd(),
+		down.NewCmd(),
```

Comment:
Do we need a `down` subcommand or can we make `up olm`/`olm up` work like `up local`, where we could create the resources, wait for the operator to run, tail the logs, and wait for the user to terminate the process. Then we could catch the signal, and tear everything down cleanly.

One problem with that approach (and there are probably others, e.g. scorecard use cases?) would be the user using SIGKILL, which cannot be caught and handled.

Thoughts?

Reply:
I imagine using `operator-sdk alpha olm up` to deploy an operator for both production and test purposes, so IMO having a process that doesn't return gives a test-only feel. `down` is for convenience, and likely won't/shouldn't be used in production.

Reply:
I think this needs more discussion regardless, since there are a few use cases to consider.